tests/test_integration.py

```diff
@@ -1,34 +1,85 @@
 import json
-import os
 import pytest
 import subprocess
-import warnings
 
 from click.testing import CliRunner
+from packaging import version
 from pathlib import Path
-from ruamel.yaml import YAML
-from unittest import TestCase
-from unittest.mock import Mock, patch
 
 from conda_vendor.conda_vendor import (
-    create_vendored_dir,
-    hotfix_vendored_repodata_json,
-    solve_environment,
-    _reconstruct_repodata_json,
+    _generate_lock_spec,
+    _get_conda_platform,
+    create_repodata_json,
     download_packages,
+    solve_environment,
+    vendor,
 )
 
-# vendor command
-from conda_vendor.conda_vendor import vendor
 
-# use python_main_defaults_environment fixture
-def test_vendor_dry_run(python_main_defaults_environment):
+environments = {
```

Comment:
If this dictionary is used only for certain tests, you could write a pytest class and save the dictionary as a class member variable to encapsulate it with the implementation. But not critical.

Reply:
Why bother?  I am leaving it as a fixture so that if I write future tests I can use it.  I don't see the value in hiding it in a class.